backend/Controllers/SubtireddController.cs

```diff
@@ -25,7 +26,7 @@ public async Task<IActionResult> Create([FromBody] CreateSubtireddModel model)
         {
             await using (tireddDbContext)
             {
-                var existingSubtiredd = await tireddDbContext.Subtiredds.SingleAsync(s => s.Name == model.Name);
+                var existingSubtiredd = await tireddDbContext.Subtiredds.FirstOrDefaultAsync(s => s.Name == model.Name);
```

Comment:
```suggestion
                var existingSubtiredd = await tireddDbContext.Subtiredds.SingleOrDefaultAsync(s => s.Name == model.Name);
```

W razie anomalii z kilkoma encjami o tej samej nazwie to nam wyrzuci exception, co zapobiegnie ukrytym błędom.